extension/agenthealth/handler/stats/agent/agent.go

```diff
@@ -118,3 +159,39 @@ type StatsConfig struct {
 	// UsageFlags are the usage flags to set on start up.
 	UsageFlags map[Flag]any `mapstructure:"usage_flags,omitempty"`
 }
+
+var StatusCodeOperations = []string{ // all the operations that are allowed
+	"DescribeInstances",
+	"DescribeTags",
+	"DescribeVolumes",
+	"DescribeContainerInstances",
+	"DescribeServices",
+	"DescribeTaskDefinition",
+	"ListServices",
+	"ListTasks",
+	"CreateLogGroup",
+	"CreateLogStream",
+}
+
+// NewStatusCodeOperationsFilter creates a new filter for allowed operations and status codes.
+func NewStatusCodeOperationsFilter() OperationsFilter {
```

Comment:
Keep these `OperationsFilter ` functions close to the type definition (move `StatsConfig` down)